src/ros_main.cpp

```diff
@@ -65,6 +67,7 @@ bool parse_args(ProgArgs &args)
   ros::param::param(PREFIX_ARG, args.prefix, std::string());
   ros::param::param(BASE_FRAME_ARG, args.base_frame, args.prefix + "base_link");
   ros::param::param(TOOL_FRAME_ARG, args.tool_frame, args.prefix + "tool0_controller");
+  ros::param::param(TCP_LINK_ARG, args.tcp_link, args.prefix + "ee_link");
```

Comment:
Could we use `tool0` for this? `ee_link`'s pose does not correspond with the UR controller's local coordinate frame in the `ur_description` xacros.

Reply:
Argh, @gavanderhoorn is right. I mixed up the two frames in the commit.
`args.prefix+ "tool0"` is the default I was going for.
Could you please change it @Zagitta ?